physics/docs/pdftxt/MYNN_EDMF.txt

```diff
@@ -28,35 +28,46 @@ model components (i.e., radiation). A description of the changes to the MYNN sch
 \cite https://doi.org/10.25923/n9wm-be49.
 
 \section intra_mynnpbl Intraphysics Communication
-- Cloud-Radiation Interaction
-\sa \ref SGSCLOUD_page
+- \ref SGSCLOUD_page
 
 The SGS clouds produced by the MYNN-EDMF scheme are coupled to the longwave and shortwave radiation schemes
```

Comment:
@mzhangw I suggest stating that it's the RRTMG radiation LW/SW schemes.

Reply:
It's probably more honest to say so, but I think it's preferable to keep this coupling work in a single interstitial module. I don't think it was necessary to add similar procedures to RRTMGP.

Reply:
@joeolson42 GP doesn't use these SGS modules, it treats clouds differently, so I suggest adding RRTMG in here to avoid this confusion. 
We should touch base on SGS cloud coupling in RRTMGP at some point soon...

Reply:
GFS_v16 and P8 do not use SGS too. SGSrad is only used in RRFS family suites and bundled with MYNN.